EnglishFilter/sections/specific.txt

```diff
@@ -13985,6 +13984,7 @@ iwatchgameofthrones.net##.the-content > p > a[href^="http://bit.ly/"]
 msn.com##.todayshowcasead
 tempr.email##.TopBanner
 ada.org,tempr.email,techcentral.ie##.topBanner
+zipextractor.app,falkirkherald.co.uk,printscreenshot.com,indiansexxxtube.com###topBanner
```

Comment:
How to reproduce it? I don't see leftover on my end.

Reply:
Sorry, I was wrong.